build.sbt

```diff
@@ -151,5 +166,17 @@ lazy val docs = project
     docusaurusCreateSite := docusaurusCreateSite.dependsOn(Compile / unidoc).value,
     docusaurusPublishGhpages := docusaurusPublishGhpages.dependsOn(Compile / unidoc).value
   )
+  .settings(macroDefinitionSettings)
   .dependsOn(coreJVM, experimentalJVM)
   .enablePlugins(MdocPlugin, DocusaurusPlugin, ScalaUnidocPlugin)
+
+lazy val examples =
```

Comment:
@jorge-vasquez-2301 @adamgfraser should examples be published? I would expect that's not the case. If my expectation is in place, please disable publishing. Otherwise, please ignore my comment.

P. S. amazing addition!

Reply:
@mijicd We shouldn't be. Good catch!

Reply:
Thanks for noticing this @mijicd ! 🙂 I'll create a PR

Reply:
Here's the PR https://github.com/zio/zio-prelude/pull/707